### PR TITLE
Add consent that data are under CDLA-Permissive-2.0 license.

### PR DESCRIPTION
**Pull request by xsuchy**
Fixes: #25

### CODE REVIEW

File: frontend/src/app/contribute.cljs [hunk 1]
```diff
@@ -177,7 +177,10 @@
                 :on-change #(on-how-to-fix-textarea-change (.-target %))}]]
 
    [:div {}
-    [:label {:class "form-label"} "Ready to submit the results?"]
+    [:label {:class "form-label"} "Ready to submit the results?"
```
Comment (FrostyX) [inline frontend/src/app/contribute.cljs:180]: You are missing a space after the question mark

Comment (xsuchy) [reply frontend/src/app/contribute.cljs]: Fixed

**Review by FrostyX — APPROVED** [review]
Thank you very much :-)

**Review by TomasTomecek — APPROVED** [review]
thank you!

**Pull request merged**